extension/package.json

```diff
@@ -1383,18 +1378,18 @@
     "viewsWelcome": [
       {
         "view": "dvc.views.actions",
-        "contents": "[$(beaker) Show Experiments](command:dvc.showExperiments)\n[$(graph-scatter) Show Plots](command:dvc.showPlots)\n[$(play) Run Experiment](command:dvc.runExperiment)",
-        "when": "!dvc.experiment.checkpoints"
```

Comment:
So we're removing the "Run Experiment" button? Isn't there enough room for four buttons?

Reply:
Also not sure about taking off the "Run Experiment" button. Isn't that button valuable for people just getting started with dvc?

Reply:
How about turning the  `+Add Pipeline` button into the `Run Experiment` ? I think the whole idea behind that button was to promote creation of the DVC config. They kinda overlap, the second one gives enough visibility already. We don't need it permanently on the side panel. WDYT folks?

Reply:
IMO it was nice to be able to run an experiment without having the experiments table open (there is also a run button available at the top of that webview). It might be just me using the product as a developer, but sometimes just having the plots webview open and running an experiment without having to switch webview is welcomed.

Reply:
> So we're removing the "Run Experiment" button? Isn't there enough room for four buttons?

The 4th button is hidden by default.

> Also not sure about taking off the "Run Experiment" button. Isn't that button valuable for people just getting started with dvc?

There is value there for anyone with the view open but there are plenty of ways to run an experiment.

> How about turning the +Add Pipeline button into the Run Experiment ? I think the whole idea behind that button was to promote creation of the DVC config. They kinda overlap, the second one gives enough visibility already. We don't need it permanently on the side panel. WDYT folks?

Good idea.